README.md

```diff
@@ -52,38 +52,49 @@ Unless otherwise noted, the source code of this project is covered under Crown C
 
 The Canada wordmark and related graphics associated with this distribution are protected under trademark law and copyright law. No permission is granted to use them outside the parameters of the Government of Canada's corporate identity program. For more information, see [Federal identity requirements](https://www.canada.ca/en/treasury-board-secretariat/topics/government-communications/federal-identity-requirements.html).
 
-______________________
+---
+
 <div lang="fr">
 
-## Boîte à outils d'accessibilité numérique
+## Boîte à Outils d'Accessibilité Numérique
 
 Ce projet en est aux premiers stades de développement et de planification, toutes les informations ne sont pas disponibles dans les deux langues officielles.
 
-<!-- TO BE TRANSLATED
-
-### Quel est ce projet?
+### Quel est ce Projet?
 
+Un **espace de partage** pour les ressources créées par les employés du gouvernement du Canada (GC) à promouvoir dans le cadre de la transparence. Suite à la [Directive sur le gouvernement ouvert] du Secrétariat du Conseil du Trésor du Canada. (https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=28108).
 
+Les types d'informations relatives à l'accessibilité numérique à partager sont:
 
-### Comment ça marche?
+- les guides,
+- liens vers des guides déjà créés en ligne,
+- les meilleures pratiques,
+- des discussions,
+- informations sur le travail que nous effectuons au sein du GC.
 
+Pour obtenir de la documentation sur les groupes de travail créés pour ce projet, consultez les [Termes de référence (ToR)](.github/TERMS_OF_REFERENCE.md).
 
+### Comment ça Marche?
 
-### Qui utilisera ce projet?
+Consultez [DEVELOPMENT.md](.github/DEVELOPMENT.md) pour plus de détails.
 
+### Qui Utilisera ce Projet?
 
+Le contenu est créé pour les employés du GC, mais il peut être utilisé par toute personne travaillant dans le domaine de l'accessibilité numérique.
 
-### Quel est le but de ce projet? 
+### Rôles Principaux
```

Comment:
Sentence case

Ceci est un sous-titre comparativement à son équivalent anglophone.

```suggestion
#### Objectif du projet
```

Reply:
Ca l'air comme vous voulez dire que Roles Principaux et Objectif du projet sons des synonymes. Key roles are the people who are using/collaborating on the project. 

I have added an extra for the next update # for Roles Principaux.